compiler/optimizations/loopInvariantCodeMotion.cpp

```diff
@@ -1210,7 +1258,10 @@ static long licmFn(FnSymbol* fn) {
 
 void loopInvariantCodeMotion(void) {
 
-  if(fNoloopInvariantCodeMotion) {
+  // compute array element alias sets
+  addNoAliasSets();
```

Comment:
I stopped adding compiler passes because it requires you to do something to the test system each time ;P

But besides that each pass runs a global garbage collection in-between and so the compiler is faster if we have fewer passes.

Happy to change the name.